app/templates/submit.hbs

```diff
@@ -250,11 +250,53 @@
                     {{/with}}
 
                     {{! ==== COI HERE ==== }}
-                    {{!-- Add logic for COI here --}}
-                    {{!-- No: Disabled Textbox with placeholder
-                        Yes: Required textarea to describe
-                    --}}
-                    {{!-- Section explaining COI here --}}
+                    {{#with _names.[6] as |name|}}
+                        {{#preprint-form-section id="author-coi-assertion" editMode=editMode name=name allowOpen=true canEdit=canEdit errorAction=(action 'error') as |hasOpened|}}
+                            {{preprint-form-header editMode=editMode node=node name=name valid=true isValidationActive=hasOpened}}
+                            {{#preprint-form-body}}
+                                <div role='radiogroup'>
+                                    <input
+                                        type="radio"
+                                        id="coiYes"
+                                        name="coi"
+                                        value="Yes"
+                                        onchange={{action "updateCoi" true}}
+                                        checked={{hasCoi}}
+                                    >
+                                    <label class='coiRadioLabel' for="coiyes">{{t "submit.body.general.yes"}}</label>
+                                    <input
+                                        type="radio"
+                                        id="coiNo"
+                                        name="coi"
+                                        value="No"
+                                        onchange={{action "updateCoi" false}}
+                                        checked={{if (and (not hasCoi) not-eq hasCoi undefined) true}}
+                                    >
+                                    <label class='coiRadioLabel' for="coiNo">{{t "submit.body.general.no"}}</label>
+                                </div>
+                                <br>
+                                {{#if hasCoi}}
+                                    <label for='coiStatement'>Describe: <span class='required' /></label>
+                                    {{validated-input id='coiStatement' inputType='textarea' model=model valuePath='conflictOfInterestStatement' documentType=currentProvider.documentType value=model.conflictOfInterestStatement}}
+                                {{else if (and (not hasCoi) (not-eq hasCoi undefined))}}
+                                    <div class='form-group'>
+                                        <input type="text" class='form-control' disabled={{true}} placeholder={{t "submit.body.conflict_of_interest.placeholder"}}>
+                                    </div>
+                                {{/if}}
+                                <div class="assertionDescription">
```

Comment:
this can get an ID so that the `<div role='radiogroup'>` can use it for its `aria-describedby` attribute